packages/gatsby-plugin-mdx/index.d.ts

```diff
@@ -0,0 +1,43 @@
+declare module "gatsby-plugin-mdx" {
```

Comment:
we probably want to use a typing file like
```
import * as React from "react"

export interface MDXRendererProps {
  scope?: any
  components?: {
    [key: string]: React.ComponentType<any>
  }
  children: string
}

export class MDXRenderer extends React.Component<MDXRendererProps> {}
```

Reply:
@wardpeet done. Thanks 🙂